opensrp-reveal/src/main/java/org/smartregister/reveal/fragment/FamilyProfileMemberFragment.java

```diff
@@ -0,0 +1,55 @@
+package org.smartregister.reveal.fragment;
+
+import android.os.Bundle;
+import android.view.View;
+
+import org.smartregister.commonregistry.CommonPersonObjectClient;
+import org.smartregister.family.fragment.BaseFamilyProfileMemberFragment;
+import org.smartregister.family.model.BaseFamilyProfileMemberModel;
+import org.smartregister.family.presenter.BaseFamilyProfileMemberPresenter;
+import org.smartregister.family.util.Constants;
+import org.smartregister.reveal.R;
+
+public class FamilyProfileMemberFragment extends BaseFamilyProfileMemberFragment {
+
+    public static BaseFamilyProfileMemberFragment newInstance(Bundle bundle) {
+        Bundle args = bundle;
+        BaseFamilyProfileMemberFragment fragment = new FamilyProfileMemberFragment();
```

Comment:
By assigning this to `BaseFamilyProfileMemberFragment` instead of `FamilyProfileMemberFragment` won't you lose any unique methods defined only in `FamilyProfileMemberFragment`?

Reply:
Changed to Child Fragment